k8s/distributed-chroma/templates/frontend-service.yaml

```diff
@@ -51,6 +51,10 @@ spec:
               {{ .Values.frontendService.logServiceHost }}
             - name: CHROMA_LOGSERVICE_PORT
               {{ .Values.frontendService.logServicePort }}
+            - name: CHROMA_FRONTEND_SERVICE__MY_MEMBER_ID
```

Comment:
`HOSTNAME` is set in the pods to the name of the pod.  I'd use that.

Reply:
As an env? Is that a k8s default thing? Can you link to docs - I didn't know that.

Reply:
I didn't either but tested it and it resulted in the same thing as `metadata.name`

Reply:
im fine with anything here fwiw.